webpack.config.js

```diff
@@ -21,8 +21,13 @@ const config = {
         about: { import: 'client/about.html', data: { title: 'LB Radio' } }
       },
       js: {
-        filename: 'main.js'
```

Comment:
the `filename` is not a path to source file.
It is the `filename template` to give the output filename based on source base filename.
Also, use it as `filename: 'js/[name].[contenthash:8].js',`  will be generate => dist/js/main.1234abcd.js (hashed name),
where `[name]` will be automatically replaced with the source base filename, e.g. `main` or `index`